src/main/java/gregtech/common/tileentities/machines/multi/GT_MetaTileEntity_HeatExchanger.java

```diff
@@ -23,8 +23,9 @@
 import net.minecraftforge.fluids.FluidRegistry;
 import net.minecraftforge.fluids.FluidStack;
 
+@SuppressWarnings("SameReturnValue")
 public class GT_MetaTileEntity_HeatExchanger extends GT_MetaTileEntity_MultiBlockBase {
-    public static float penalty_per_config = 0.015f;  // penalize 1.5% efficiency per circuitry level (1-25)
+    public static final float penalty_per_config = 0.015f;  // penalize 1.5% efficiency per circuitry level (1-25)
```

Comment:
+

Reply:
this is a constant value that doesn't ever get changed. can stay final